bus-mapping/src/evm/opcodes/callop.rs

```diff
@@ -435,6 +431,33 @@ impl<const N_ARGS: usize> Opcode for CallOpcode<N_ARGS> {
     }
 }
 
+fn write_memory_word_ops(
+    state: &mut CircuitInputStateRef,
+    exec_step: &mut ExecStep,
+    call_id: usize,
+    offset: usize,
+    length: usize,
+    data: &[u8],
+) {
+    let shift = offset % 32;
+    let slot = offset - shift;
+
+    let mut i = 0;
+    while i < length {
+        let mut slot_bytes: [u8; 32] = [0; 32];
+        slot_bytes.clone_from_slice(&data[i..length.min(i + 32)]);
```

Comment:
I see `clone_from_slice` function will panic if the two slices have different lengths. do you test length < 32 case?

Reply:
Fixed in commit [8ecae1c](https://github.com/scroll-tech/zkevm-circuits/pull/485/commits/8ecae1c3a9cb90902e9aafdf883318c7af714c55).